### PR TITLE
Delete feed on disconnect/uninstall.

### PR DESCRIPTION
**Pull request by message-dimke**
### Changes proposed in this Pull Request:

Delete feed instead of deactivating it on disconnect/uninstall action.

### Changelog entry

> Update - Delete feed instead of deactivate.

### CODE REVIEW

File: src/Feeds.php [hunk 1]
```diff
@@ -354,7 +354,27 @@ public static function enabled_feed( string $feed_id ): bool {
 	public static function disable_feed( string $feed_id ): bool {
```
Comment (budzanowski) [inline src/Feeds.php]: It looks like we should able to remove this method now since we are not using diable_feed.

**Review by budzanowski — CHANGES_REQUESTED** [review]
Nice. I like that you have fixed the parameters order!

**Comment by budzanowski** [comment]
Looks good. We can merge. Please just remove the disable_feed in case we don't need it.

File: src/Feeds.php (hunk 1 above)
Comment (message-dimke) [reply src/Feeds.php]: Removing enable/disable feed methods according to logic changes (replacing with delete feed method).

**Pull request merged**